websites/R/RhysBOT/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+	clientId: "658765364439810048"
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/yAhhPzj.png",
+			details: "Exploring",
+			startTimestamp: Date.now(),
```

Comment:
By setting this _inside_ the event, it means the timestamp will constantly be updating